### PR TITLE
Qterminal bookmarks blog post

### PR DESCRIPTION
**Pull request by stefonarch**

### CODE REVIEW

File: _posts/2022-03-15-blog-qterminal-bookmarks.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+title: QTerminal Bookmarks reloaded
+slug: qterminal-bookmarks-reloaded
+date: mar 15, 2022, 14:54:49
+promoted: true
+categories: blog
+---
+
+QTerminal had always had a quite unique "bookmarks" feature but the setup was not quite intuitive: the file had
+to be created first, it had a hardcoded section containing not very useful bookmarks, filtering never worked
+ and so on. Probably few users used it, as some issue never were reported.
```
Comment (tsujan) [inline _posts/2022-03-15-blog-qterminal-bookmarks.md:12]: The content is very good. Just a few corrections:

"as some issue never were reported." → "as some issues were never reported."

File: _posts/2022-03-15-blog-qterminal-bookmarks.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ Now - among many other bugfixes in QTerminal and his dropdown twin -
```
Comment (tsujan) [inline _posts/2022-03-15-blog-qterminal-bookmarks.md:13]: "QTerminal and his dropdown twin" → "QTerminal and its dropdown twin"

File: _posts/2022-03-15-blog-qterminal-bookmarks.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ we dedicated some time to make this feature more attractive and usable to users.
+
+ It can be used in a similar way as <code>~/.bash_aliases</code> - a real easy way to use
```
Comment (tsujan) [inline _posts/2022-03-15-blog-qterminal-bookmarks.md:16]: "a real easy way"  → "a really easy way"

File: _posts/2022-03-15-blog-qterminal-bookmarks.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  longer commands or hard ones to remember. Commands can be ordered in groups.
```
Comment (tsujan) [inline _posts/2022-03-15-blog-qterminal-bookmarks.md:17]: "longer commands or hard ones to remember" → "longer commands or those that are hard to remember"

**Comment by stefonarch** [comment]
Thanks!

**Pull request merged**